ThinBridge/Chronos
# Remove IE from redirect examples

# Which issue(s) this PR fixes:

> Describe issue number. If issue doesn't exist, fill N/A.

https://github.com/ThinBridge/Chronos-SG/issues/306

# What this PR does / why we need it:

> Explain the detail of the problem.

Remove IE from the OnRedirect template as IE support has ended.

# How to verify the fixed issue:

> Describe the following information to help that the tester able to do it.

## The steps to verify:

1. Start Chronos
2. Open Tools -> Preferences -> Redirection
3. Check "Enable URL redirection"  to enable [Open Script Editor Window] button
4. Click it to open Script Editor Window
5. Click [Help] to see the result, then click [Close]
6. Click [Open Script Editor Window] again
7. Click [Function OnRedirect] to see the result

## Expected result:

> Describe the obvious situation to verify.

* At step 5, confirm that
  * No IE example is among "For the case" lines at the top
  * The example at the bottom has a line stating OnRedirect="Edge" instead of IE
* At step 7, confirm that the template appeared has no OnRedirect="IE"
